socialdistribution/app/views.py

```diff
@@ -10,6 +11,10 @@ def register(request):
     if request.method == 'POST':
         form = RegisterForm(request.POST)
         if form.is_valid():
+            # https://www.youtube.com/watch?v=q4jPR-M0TAQ&list=PL-osiE80TeTtoQCKZ03TU5fNfx2UY6U4p&index=6 
+            # Will give a notification when register successfully 
+            username = form.cleaned_data.get('username')
+            messages.success(request,f'Account created for {username}!')
```

Comment:
Could we change this to something like `f'Request to register account {username} has been submitted!'`?

Accounts have to be approved by the server admin before they're active and can log in, so it might be better to let them know that the request was just submitted at this point.